src/pages/login.js

```diff
@@ -0,0 +1,273 @@
+    borderRadius: "10px",
+    backgroundColor: theme.palette.primary.main,
+    color: 'white',
+    fontSize: "24px",
+    textAlign: "center",
+    textTransform: "uppercase",
+    padding: "10px 0px 10px 0px",
+    boxShadow: "none",
+
+    width: "100%",
+    height: "80px",
+
+    "&:hover": {
+      borderWidth: "3px",
+      backgroundColor: "transparent",
+      color: theme.palette.primary.main,
+      boxShadow: "none",
+    },
+  },
+  rememberMe: {
+    fontSize: 18,
+    color: "#969696",
+  },
+
+  icon: {
+    borderRadius: 3,
+    borderStyle: "solid",
+    width: 16,
+    height: 16,
+    marginRight: "3px",
+    "input:hover ~ &": {
+      backgroundColor: theme.palette.type === "dark" ? "#6f6f6f" : "#e8e8e8",
+    },
+  },
+  checkedIcon: {
+    backgroundColor: "#21CE99",
+    borderStyle: "solid",
+    borderColor: "#1AA47A",
+    "input:hover ~ &": {
+      backgroundColor: "#21CE99",
+    },
+  },
+}));
+
+function StyledCheckbox(props) {
+  const classes = useStyles();
+  return (
+      <Checkbox
+          className={classes.root}
+          disableRipple
+          color="primary"
+          checkedIcon={<span className={clsx(classes.icon, classes.checkedIcon)} />}
+          icon={<span className={classes.icon} />}
+          inputProps={{ "aria-label": "decorative checkbox" }}
+          {...props}
+      />
+  );
+}
+
+function Login(props) {
+  const classes = useStyles();
+  const {register, handleSubmit} = useForm()
+  let history = useHistory();
+
+  const onSubmit = (data) => {
+    var authenticationDetails = new AmazonCognitoIdentity.AuthenticationDetails({
+      Username: data.email,
+      Password: data.password,
+    });
+    var poolData = {
+        UserPoolId: 'ap-southeast-2_9hmZTBuah', // Your user pool id here
+        ClientId: '3u8dhsro0i2igvhuvbl461eenv', // Your client id here
+    };
+    var userPool = new AmazonCognitoIdentity.CognitoUserPool(poolData);
+
+    var user = new AmazonCognitoIdentity.CognitoUser({
+        Username: data.email,
+        Pool: userPool,
+    });
+
+  // History to redirect to revise page
+
+  // TODO: Add user to Group (Basic Auth user)
+
+  
+  
+  user.authenticateUser(authenticationDetails, {
+    onSuccess: function(result) {
+      console.log(result)
+      console.log("Token : " + result.getIdToken().getJwtToken());
+      var accessToken = result.getIdToken().getJwtToken();
+      
+      // Use Access token to init user with AWS Creds
+      AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+        IdentityPoolId: 'ap-southeast-2:4ca5f7a7-14e9-411d-b0f0-0c40fd8c67b2',
+        Logins: {
+            'cognito-idp.ap-southeast-2.amazonaws.com/ap-southeast-2_9hmZTBuah': accessToken
+        }
+      });
+      AWS.config.credentials.get(function(err){
+        if (err) {
+            alert(err);
+        }
+      });  
+
+      history.push("/revise");
+
+      
+
+        // AWS.config.credentials = new AWS.CognitoIdentityCredentials({
```

Comment:
whats this commented code for?

Reply:
Hitarth wrote this code so I think he can explain it better. If he thinks it's not necessary I can remove them